checks/raw/pinned_dependencies.go

```diff
@@ -744,6 +744,18 @@ var validateGitHubActionWorkflow fileparser.DoWhileTrueOnFileContent = func(
 		if len(fileparser.GetJobName(job)) > 0 {
 			jobName = fileparser.GetJobName(job)
 		}
+
+		if job.WorkflowCall != nil && job.WorkflowCall.Uses != nil {
+			//nolint:lll
+			// Check whether this is an action defined in the same repo,
+			// https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#referencing-an-action-in-the-same-repository-where-a-workflow-file-uses-the-action.
+			if !strings.HasPrefix(job.WorkflowCall.Uses.Value, "./") {
+				// This can be deduplicated:
```

Comment:
the comment is no longer applicable since you did deduplicate

Reply:
https://github.com/ossf/scorecard/pull/4681/commits/ab22a6edffbc305d6c4d000c314e15add4e00ffc